src/screens/SettingsSheet/components/GoogleAccountSection.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from 'react';
+import {
+  getGoogleAccountUserData,
+  GoogleDriveUserData,
+  logoutFromGoogleDrive,
+} from '@/handlers/cloudBackup';
+import ImageAvatar from '@/components/contacts/ImageAvatar';
+import { showActionSheetWithOptions } from '@/utils';
+import * as i18n from '@/languages';
+import {
+  clearAllWalletsBackupStatus,
+  updateWalletBackupStatusesBasedOnCloudUserData,
+} from '@/redux/wallets';
+import { useDispatch } from 'react-redux';
+import Menu from './Menu';
+import MenuItem from './MenuItem';
+
+export const GoogleAccountSection: React.FC = () => {
+  const dispatch = useDispatch();
+  const [accountDetails, setAccountDetails] = useState<
+    GoogleDriveUserData | undefined
+  >(undefined);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    getGoogleAccountUserData()
+      .then(accountDetails => {
+        setAccountDetails(accountDetails ?? undefined);
+      })
+      .catch(e => {
+        console.log('Error while getting google account data', e);
```

Comment:
did you mean to log this to sentry?

Reply:
Unresolved cuz noticed it wasn't done yet 😛